contactaccessrights/CRM/Contactaccessrights/Helper/ContactActionsMenu/ContactAccessActionGroup.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+use CRM_HRContactActionsMenu_Component_Group as ActionsGroup;
+use CRM_Contactaccessrights_Service_ContactRights as ContactRightsService;
+use CRM_HRCore_CMSData_UserPermissionInterface as CMSUserPermission;
+use CRM_HRContactActionsMenu_Component_GroupButtonItem as ActionsGroupButtonItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_GroupTitleToolTipItem as GroupTitleToolTipItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_NoStaffTooltipItem as NoStaffToolTipItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_GenericTextItem as GenericTextItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_UserRegionsListItem as UserRegionsListItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_UserLocationsListItem as UserLocationsListItem;
+use CRM_Contactaccessrights_Component_ContactActionsMenu_UserACLGroupsListItem as UserACLGroupsListItem;
+
+class CRM_Contactaccessrights_Helper_ContactActionsMenu_ContactAccessActionGroup {
+
+  /**
+   * @var array
+   */
+  private $contactUserInfo;
+
+  /**
+   * @var ContactRightsService
+   */
+  private $contactRightsService;
+
+  /**
+   * @var array
+   */
+  private $contactACLGroups;
+
+  /**
+   * @var CMSUserPermission
+   */
+  private $CMSUserPermission;
```

Comment:
We don't start variable names with uppercase letters. Please change this to `$cmsUserPermission`